azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -627,6 +634,14 @@ public long getNumberOfAgedQueuedFlows() {
     this.webMetrics.startReporting(this.props);
   }
 
+  /**
+   * Set up and start reporting container metrics
+   */
+  private void startContainerMetrics() {
+    this.containerizationMetrics.setUp();
```

Comment:
I don't see shutdown method called from anywhere. When web server is shutting down, it should be called.